spec/fixtures/forward-proxy-server.lua

```diff
@@ -0,0 +1,116 @@
+local _M = {}
```

Comment:
what's the difference between this and the naive forward proxy fixture here: https://github.com/Kong/kong-ee/pull/2945/files#diff-a2b4ed7d776c435259b62afad583c89bd0ce9f9143d3ba4d104500e352c10f7aR16 

should we use only one implementation for tests (if they are doing the same thing)?

Reply:
Oof. There's no difference--just two developers managing to unknowingly implement the same test fixture within \*checks other commit\* 24 hours of each other. :upside_down_face: